packages/database/src/migrations/20230219232553-AddMetadataInDataTable-modifies-data.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { generateId, arrayToDbString } = require('../utilities');
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const newTypes = ['data_group', 'data_elements'];
+
+exports.up = async function (db) {
+  const query = newTypes
+    .map(t => {
+      const tupaiaMetadataDescription = `Fetches metadata for tupaia ${t.split('_').join(' ')}`;
+      const dhisMetadataDescription = `Fetches metadata for DHIS2 ${t.split('_').join(' ')}`;
+      const metadataCode = `${t}_metadata`;
+      return `
+      INSERT INTO data_table VALUES ('${generateId()}','tupaia_${metadataCode}', '${tupaiaMetadataDescription}', '{}', '{"*"}', '${metadataCode}');
+      INSERT INTO data_table VALUES ('${generateId()}','dhis_${metadataCode}', '${dhisMetadataDescription}', '{}', '{"*"}', '${metadataCode}');
```

Comment:
What's the purpose of separating Tupaia and DHIS2 metadata data-tables here? Won't they behave the exact same, since we just pass the data-element codes to data-broker (which supports Tupaia and DHIS2)?

Reply:
Yeah they are both supported, and we could use the same endpoint for them. However data broker does not accept data elements from different data service. I find it might be easier for users to avoid this exception.

Reply:
I think data-broker throws a fairly clear error message about this if it happens right? (And if not we could just improve the error messaging).
I think users should learn that they can only pull from one service type at a time, and then they can just do multiple pulls from the same data-table if they need data from multiple services 👍

Reply:
Yeah fair point👍. Will apply.